microcosm_flask/conventions/crud_adapter.py

```diff
@@ -59,9 +59,19 @@ def update_batch(self, **kwargs):
 
         """
         items = kwargs.pop("items")
+
+        def transform(item):
```

Comment:
After testing in an actual service, I discovered that we need this transform for a default/vanilla batch update.